images.CI/macos/azure-pipelines/image-generation.yml

```diff
@@ -7,6 +7,8 @@ jobs:
   variables:
   - group: Mac-Cloud Image Generation
   - group: Mac-Cloud Image Generation Key Vault
+  - name: VirtualMachineName
+    value: $(Build.BuildNumber).$(System.JobAttempt)

```

Comment:
I don't think that it is good option. With this approach we will VM name like `macOS-10.15_20200922.9_unstable.1`

Reply:
It is better to keep `_unstable` prefix at the end

Reply:
I think we can keep this variant.

Reply:
Sounds good